### PR TITLE
[Filebeat] Update `checkpoint.source_object` mapping

### LINKED ISSUE #25124: [Check Point] Field checkpoint.source_object mapped incorrectly as long
- Version: 7.11.2
- Operating System: n/a
- Steps to Reproduce: 

1. `./filebeat setup`
2. observe field mapping for `checkpoint.source_object`

According to https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk144192, Check Point module field `checkpoint.source_object` should be mapped as a string type instead of long. Events with the field can encounter a `mapper_parsing_exception` with the current template:

```
[2021-04-02T17:07:49,508][WARN ][logstash.outputs.elasticsearch][checkpoint-security][checkpoint-security] Could not index event to Elasticsearch. {:status=>400, :action=>["index", {:_id=>nil, :_index=>"checkpoint-security_filebeat-7.11.2", :routing=>nil, :pipeline=>"filebeat-7.11.2-checkpoint-firewall-pipeline"}, #LogStash::Event:0x36ebe333], :response=>{"index"=>{"_index"=>"checkpoint-security_filebeat-7.11.2-2021.04.02-000021", "_type"=>"_doc", "_id"=>"2FYgk3gB7n6nMgM0RtU0", "status"=>400, "error"=>{"type"=>"mapper_parsing_exception", "reason"=>"failed to parse field [checkpoint.source_object] of type [long] in document with id '2FYgk3gB7n6nMgM0RtU0'. Preview of field's value: 'Africa'", "caused_by"=>{"type"=>"illegal_argument_exception", "reason"=>"For input string: "Africa""}}}}}
```

### Workaround

To workaround, override the default field mapping with an additional higher order template.

```
PUT /_template/filebeat-checkpoint-source-object-override
{
  "order": 10,
  "index_patterns": [
    "filebeat-*"
  ],
  "mappings": {
    "properties": {
      "checkpoint": {
        "properties": {
          "source_object": {
            "ignore_above": 1024,
            "type": "keyword"
          }
        }
      }
    }
  }
}
```

The field will have the correct mapping when a new Filebeat index is created.

### PR DESCRIPTION
**Pull request by legoguy1000**
## What does this PR do?

Changes `checkpoint.source_object` from Long to Keyword per https://supportcenter.checkpoint.com/supportcenter/portal?eventSubmit_doGoviewsolutiondetails=&solutionid=sk144192

## Why is it important?

Field should be a string, not number

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have made corresponding change to the default configuration files
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally
```
cd x-pack/filebeat
TESTING_FILEBEAT_MODULES=checkpoint mage -v pythonIntegTest
```

## Related issues

- Closes #25124

### CODE REVIEW

File: x-pack/filebeat/module/checkpoint/firewall/_meta/fields.yml
Comment (inqueue) [inline x-pack/filebeat/module/checkpoint/firewall/_meta/fields.yml:1893]: I believe this will solve the issue!

Comment (legoguy1000) [reply x-pack/filebeat/module/checkpoint/firewall/_meta/fields.yml:1893]: ok will take out of draft.

**Comment by legoguy1000** [comment]
@andrewkroh @andrewstucki could you run the CI tests?

**Comment by andrewkroh** [comment]
run tests

**Comment by legoguy1000** [comment]
@andrewkroh I think we're good to merge unless you have any comments.

**Comment by andrewkroh** [comment]
run tests

**Comment by legoguy1000** [comment]
Looks like all the tests passed

**Comment by andrewkroh** [comment]
I suspected that this fail after it merged to master because in master I pinned the geoip database version which affected the contents of the generated files. So I updated the branch and regenerated the test output files.

**Review by andrewkroh — APPROVED** [review]

**Comment by andrewkroh** [comment]
run tests

**Pull request merged**
